rs/consensus/src/idkg/payload_builder/pre_signatures.rs

```diff
@@ -359,6 +359,7 @@ fn make_new_pre_signatures_if_needed_helper(
     let Some(pre_signatures_to_create) = chain_key_config
         .key_configs
         .iter()
+        .filter(|key_config| !matches!(&key_config.key_id, MasterPublicKeyId::VetKd(_)))
```

Comment:
This could later lead to problems, if we ever where to add another variant. I think it would be better to filter for the variants that we want to have here.

Reply:
Changed it to

```
.filter(|key_config| { matches!(&key_config.key_id, MasterPublicKeyId::Ecdsa(_) | MasterPublicKeyId::Schnorr(_)) })
```

Reply:
Resolving this conversation, because the comment was addressed.